lib/stm32/f4/rcc.c

```diff
@@ -512,10 +512,10 @@ void rcc_osc_off(enum rcc_osc osc)
 		RCC_CSR &= ~RCC_CSR_LSION;
 		break;
 	case RCC_PLLSAI:
-		RCC_CSR &= ~RCC_CR_PLLSAION;
+		RCC_CR &= ~RCC_CR_PLLSAION;
 		break;
 	case RCC_PLLI2S:
-		RCC_CSR &= ~RCC_CR_PLLI2SON;
+		RCC_CR &= ~RCC_CR_PLLI2SON;
```

Comment:
Can you fix these in the first commit in the series? they were only just introduced weren't they?